encodings/caesar/caesar.py

```diff
@@ -0,0 +1,47 @@
+import sys
+import caesar_args
+
+# All input should be uppercased
+CHARACTER_OFFSET = 65
+ALPHABET_SIZE = 26
+
+def shift_letter(letter, shift_count):
+    char_code = ord(letter) - CHARACTER_OFFSET
+    if char_code < 0:
+        print("Error, invalid letter: " + letter)
```

Comment:
Raise an exception here instead. That way a potential caller can handle it accordingly

Reply:
I'd rather just print non-alpha characters since I can see myself copy-pasting a phrase in there that may include some punctuation or something